modules/dnn/src/layers/resize_layer.cpp

```diff
@@ -52,11 +57,15 @@ class ResizeLayerImpl : public ResizeLayer
     virtual bool supportBackend(int backendId) CV_OVERRIDE
     {
 #ifdef HAVE_INF_ENGINE
-        if (backendId == DNN_BACKEND_INFERENCE_ENGINE)
+        if (backendId == DNN_BACKEND_INFERENCE_ENGINE_NN_BUILDER_2019)
         {
             return (interpolation == "nearest" && scaleWidth == scaleHeight) ||
                    (interpolation == "bilinear");
         }
+        if (backendId == DNN_BACKEND_INFERENCE_ENGINE_NGRAPH) {
+            return (interpolation == "nearest" && scaleWidth == scaleHeight && scaleWidth == 0.5) ||
```

Comment:
That means only x2 nearest neighbor downsampling is supported?

Reply:
Yes